server/schedule/store_limit.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schedule
+
+import (
+	"time"
+
+	"github.com/juju/ratelimit"
+	"github.com/pingcap/pd/server/schedule/operator"
+)
+
+// StoreLimitMode indicates the strategy to set store limit
+type StoreLimitMode int
+
+// There are two modes supported now, "auto" indicates the value
+// is set by PD itself. "manual" means it is set by the user.
+// An auto set value can be overwrite by a manual set value.
+const (
+	StoreLimitAuto StoreLimitMode = iota
```

Comment:
How about directly using `Auto` and `Manual`?

Reply:
The file is in the `schedule` package, it is too general to use the simple words `Auto` and `Manual`. 

For example,  what do you mean when you use `schedule.Auto`? So I think the prefix is necessary.